kpatch/kpatch

```diff
@@ -165,7 +165,27 @@ load_module () {
 	fi
 
 	echo "loading patch module: $module"
-	insmod "$module"
+	local i=0
+	while true; do
+		out=$(insmod "$module" 2>&1)
+		[[ -z $out ]] && break
+		echo $out 1>&2
+		[[ ! $out =~ "Device or resource busy" ]] &&
+			die "failed to load module $module"
+
+		# "Device or resource busy" means the activeness safety check
+		# failed.  Retry in a few seconds.
+		i=$((i+1))
+		if [[ $i = 5 ]]; then
```

Comment:
Okay, like I said, it just gets confusing, especially with ```-gt``` vs ```>```.  I did test this in a shell and you're right, the string comparison will match.